pytest_remotedata/disable_internet.py

```diff
@@ -41,7 +60,7 @@ def new_function(*args, **kwargs):
                 return original_function(*args, **kwargs)
             host = args[1][0]
             addr_arg = 1
-            valid_hosts = ('localhost', '127.0.0.1', '::1')
+            valid_hosts = set(['localhost', '127.0.0.1', '::1'])
```

Comment:
Note: I use `set` here because different host names sometimes resolve to the same IPs.